lib/try.js

```diff
@@ -1,17 +1,75 @@
 var express = require('express');
 var request = require('request');
+var streamBuffers = require("stream-buffers");
 
 module.exports = function() {
   var proxy = new express.Router();
   
-  proxy.all('/', function(req, res) {
+  proxy.all('/', function( req, resp ) {
+    var requestedUrl = getRequestedUrl( req );
+    enrichResponseWithRequestHeaders( req, resp );
+    var requestBuffer = new streamBuffers.WritableStreamBuffer({
+      initialSize: (100 * 1024),      // start as 100 kilobytes.
+      incrementAmount: (10 * 1024)    // grow by 10 kilobytes each time buffer overflows.
```

Comment:
Right, seems like a premature optimization, and indeed, is a default :-)

What about introducing a config.js to express that this is subject of optimization in the future?